src/property-filter/styles.scss

```diff
@@ -98,6 +116,17 @@ $operator-field-width: 120px;
     flex-grow: 2;
   }
 
+  &-multiselect-wrapper {
+    position: relative;
+    block-size: awsui.$size-vertical-input;
+    min-inline-size: 200px;
+
+    &-inner {
+      position: absolute;
```

Comment:
Why do you need to have an extra inner wrapper with absolute position? A quick check removing this rule in the browser made no difference

Reply:
That is to avoid the input growing when more options are added

https://github.com/user-attachments/assets/1a2fcdbc-6acf-4a49-9ac7-53efe0da23fe